### PR TITLE
Update procedure-project-management.md

### PR DESCRIPTION
**Pull request by irogers**
Updates to analytics workflow and folder structure

### CODE REVIEW

File: content/internal/procedure-project-management.md [hunk 1]
```diff
@@ -57,27 +85,24 @@ At the top level a project should contain the following:
...
 This is what should go into each of those folders:
 
 - `data/` — raw data files
 - `doc/` — background documentation (static)
-- `notebooks/` — analyses
-- `python/` — Python code
+- `notebooks/` — experimental analysis
 - `R/` — R code (normally the contents of a package)
-- `reports/` — generated reports
-- `scripts/` — R or Python scripts
```
Comment (datawookie) [inline content/internal/procedure-project-management.md:75]: I think we should retain the `scripts/` folder. On R projects this is where I would normally keep my non-package scripts. My thinking on this structure is that it's a superset of what is actually used in practice. So if it doesn't make sense for a particular project to have a `scripts/` folder then just leave it out.

File: content/internal/procedure-project-management.md [hunk 2]
```diff
@@ -57,27 +85,24 @@ At the top level a project should contain the following:
...
 └── src
 ```
 
 This is what should go into each of those folders:
 
 - `data/` — raw data files
 - `doc/` — background documentation (static)
-- `notebooks/` — analyses
-- `python/` — Python code
+- `notebooks/` — experimental analysis
```
Comment (datawookie) [inline content/internal/procedure-project-management.md]: Plural "analyses"?

File: content/internal/procedure-project-management.md [hunk 3]
```diff
@@ -57,27 +85,24 @@ At the top level a project should contain the following:
...
 - `R/` — R code (normally the contents of a package)
-- `reports/` — generated reports
-- `scripts/` — R or Python scripts
+- `reports/` — `.R` or  `.py` files used to generate analyses for reports 
 - `src/` — Python code (normally the contents of a module)
 
 ### Notebooks
 
-The notebooks should be named according to the following convention: `<initials>_<YYYYMMDD>_<title>`.
+- Notebooks are an experimental area, and are the domain of the individual who creates them. These documents should be named according to the following convention: `<initials>_<YYYYMMDD>_<title>`
```
Comment (datawookie) [inline content/internal/procedure-project-management.md]: Terminal period.

File: content/internal/procedure-project-management.md [hunk 4]
```diff
@@ -57,27 +85,24 @@ At the top level a project should contain the following:
...
+- Scripts in `reports/` are the responsibility of the team to mantain
```
Comment (datawookie) [inline content/internal/procedure-project-management.md]: Terminal period.

**Comment by datawookie** [comment]
Hey @irogers, just added a few minor comments. Otherwise I think this looks good. Thanks, Andrew.

File: content/internal/procedure-project-management.md [hunk 5]
```diff
@@ -46,6 +46,34 @@ Where necessary, include the following information on the Asana card for a task
...
+    - Identify analyses to perform
+3. Analysis
+4. Submission of Draft Report
+5. Feedback & revisions
+6. Submission of Final report
+7. Discussion of the report (what is actionable and why)
+
+### Report Format
+
+- Always send report with Exegetic letterhead on first page. Maybe we should rather have a “cover page”?
```
Comment (datawookie) [inline content/internal/procedure-project-management.md:72]: Which do you think? I reckon that an Exegetic cover page would be more flexible than a letterhead. Cover page would have slots for report title, author(s) and date.

Comment (irogers) [reply content/internal/procedure-project-management.md:72]: Not too sure. Might be a bit much for collaborative documents. If we need to add branding I would personally put something like faded logos of the client and exegetic at the footer

**Pull request merged**